packages/api/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+services:
+  # web:
```

Comment:
This is commented out but should build and start the express web API. 
Currently fails on M1 mac because of a binary target error.